setup.py

```diff
@@ -12,7 +12,7 @@
       install_requires=[
           "pendulum==2.1.2",
```

Comment:
Also, the latest version of Pendulum is 3.1.0. If you're upgrading other libraries to their latest versions, why not this one as well?